### PR TITLE
Add option for increased onnx logging

### PR DESCRIPTION
**Pull request by CoderHam**

### CODE REVIEW

File: src/backends/onnx/loader.cc [hunk 1]
```diff
@@ -48,8 +49,13 @@ OnnxLoader::Init()
     OrtEnv* env;
     // If needed, provide custom logger with
     // ort_api->CreateEnvWithCustomLogger()
-    OrtStatus* status =
-        ort_api->CreateEnv(ORT_LOGGING_LEVEL_WARNING, "log", &env);
+    OrtStatus* status;
+    if (LOG_VERBOSE_IS_ON(1)) {
```
Comment (deadeyegoodwin) [inline src/backends/onnx/loader.cc]: I thought you were only going to turn on ONNX verbose at level 2 or higher?

**Comment by deadeyegoodwin** [comment]
@CoderHam  I don't see a CI run for this PR

File: src/backends/onnx/loader.cc (hunk 1 above)
Comment (CoderHam) [reply src/backends/onnx/loader.cc]: Done

**Comment by CoderHam** [comment]
> @CoderHam I don't see a CI run for this PR

Started with above log level changes.

**Pull request merged**